### ISSUE TITLE
would it make sense to allow `frame_memory` into the future?

### ISSUE THREAD

**Issue by joshinils** [tfaehse/DashcamCleaner#24]
what is the reason for this setting?

I can only think of needing to also blur the same position a frame around the actual detection in-case the next frame does not detect it or detects poorly.

in that case blurring the frames around the detection makes more sense i think than only in one time-direction.

at least it could be an option if one needs it and it is easy to implement

**Comment by tfaehse** [comment]
It's fairly specific, but there are use cases (granted, mostly for cars...) where I found it really useful. When recording large, stationary scenes from a stationary car changing light conditions or partial occlusions may cause detections to jitter a bit. With this setting enabled those scenes look much nicer.

Blurring around the area would be a nice idea, but since there's no memory (or logic in that regard) I can't tell if a detection disappeared from one frame to the next because it moved, because it was hidden or because it's somewhere else in the frame. I've experimented with tracking objects across frames, but so far performance really doesn't look good.
